splitio/api/impressions.py

```diff
@@ -91,14 +94,16 @@ def flush_impressions(self, impressions):
         :type impressions: list
         """
         bulk = self._build_bulk(impressions)
+        start = int(round(time.time() * 1000))
```

Comment:
The expression `int(round(time.time() * 1000))` is being used in multiple places. 
If you think it might make your code simpler, consider defining some utility function.

Example in JS: https://github.com/splitio/javascript-commons/blob/development/src/utils/timeTracker/timer.ts

Reply:
will do.